### PR TITLE
Recover hormuud week 6

### PR DESCRIPTION
**Pull request by IsaackMwenda**

### CODE REVIEW

File: src/translate_rapid_pro_keys.py [hunk 1]
```diff
@@ -113,8 +126,12 @@ def remap_radio_shows(cls, user, data, coda_input_dir):
         :param coda_input_dir: Directory to read coded coda files from.
         :type coda_input_dir: str
         """
-        # No implementation needed yet, because no flow is yet to go wrong in production.
-        pass
+        # Correct drama week 6 messages recovered from the Hormuud downtime issue
+        cls._remap_radio_show_by_time_range(
+            user, data, "received_on", "rqa_s01e06_raw",
+            isoparse("2019-05-27T00:00:25+03:00"),
```
Comment (as2388) [inline src/translate_rapid_pro_keys.py]: Is there someone who could double check these dates?

File: generate_outputs.py [hunk 2]
```diff
@@ -156,6 +161,21 @@
         with open(path, "r") as f:
             messages_datasets.append(TracedDataJsonIO.import_json_to_traced_data_iterable(f))
 
+    # TODO Update this to read from config - RecoveryCSVURLs
+    recovery_datasets = []
+    if pipeline_configuration.recovery_csv_urls is None:
```
Comment (as2388) [inline generate_outputs.py]: I see no reason to include the if on the recovery_csv_urls if you're not using them.

I would keep the TODO, but drop everything except the body of the else.

Comment (IsaackMwenda) [reply generate_outputs.py]: Your are right. Updated thanks!

File: docker-run-generate-outputs.sh [hunk 3]
```diff
@@ -19,14 +19,15 @@ while [[ $# -gt 0 ]]; do
...
+if [[ $# -ne 24 ]]; then
     echo "Usage: ./docker-run-generate-outputs.sh
     [--profile-cpu <profile-output-path>]
     <user> <pipeline-configuration-file-path> <google-cloud-credentials-file-path> <phone-number-uuid-table-path>
-    <s01e01-input-path> <s01e02-input-path> <s01e03-input-path> <s01e04-input-path> <s01e05-input-path> <s01e06-input-path>
-    <s01e07-input-path> <s01mag03-input-path> <s01mag04-input-path> <s01mag05-input-path>
-    <s01-demog-input-path> <s01-follow-up-w2-input-path> <prev-coded-dir> <json-output-path>
-    <icr-output-dir> <coded-output-dir> <messages-output-csv> <individuals-output-csv> <production-output-csv>"
+    <s01e01-input-path> <s01e02-input-path> <s01e03-input-path> <s01e04-input-path> <s01e05-input-path>
+    <s01e06-input-path> <s01e06_hormuud_recovered_input_path> <s01e07-input-path> <s01mag03-input-path>
```
[the remark this answers is not in the document]
Comment (IsaackMwenda) [reply docker-run-generate-outputs.sh]: Updated thanks!

File: src/translate_rapid_pro_keys.py (hunk 1 above)
Comment (IsaackMwenda) [reply src/translate_rapid_pro_keys.py]: Sorted the csv by date picked the minimum and maximum dates. The log on number of remapped messages matches the no of loaded messages. Episode 6 dates double checked with RDA.

**Pull request merged**